flare/framework/computation_task.py

```diff
@@ -44,30 +45,35 @@ def __init__(self,
         self.name = name
         self.hp = hyperparas
         if model_dir == "":
-            self.model_file = ""
+            self.model_dir = ""
         else:
             os.system("mkdir -p " + model_dir)
-            self.model_file = model_dir + "/" + name
+            self.model_dir = model_dir
         self.alg = algorithm
         self.optim = optim.RMSprop(
             self.alg.model.parameters(), lr=hyperparas["lr"])
         self._cdp_args = kwargs
         self._cdp = None
-        ## if model_file is not empty, then we load an init model
-        if self.model_file != "":
-            models = glob.glob(self.model_file)
+        ## if model_dir is not empty, then we load an init model
+        if self.model_dir != "":
+            models = glob.glob(self.model_dir + "/*")
             if models:
                 ## for now, we take the most recent model
-                most_recent_model_file = sorted(models)[-1]
+                most_recent_model_pass = sorted(models)[-1]
```

Comment:
it might be better to sort models based on the modification time, e.g. sorted(models, key=os.path.getmtime)?